src/Symfony/Component/EventDispatcher/Debug/WrappedListener.php

```diff
@@ -118,4 +119,24 @@ public function __invoke(object $event, string $eventName, EventDispatcherInterf
             $this->stoppedPropagation = true;
         }
     }
+
+    private function getNameFromArray(array $listener): ?string
+    {
+        if ($listener[0] instanceof \Closure) {
+            try {
+                foreach ((new \ReflectionFunction($listener[0]))->getAttributes(Closure::class) as $attribute) {
```

Comment:
What about going with `\Closure::class` actually?
Putting an attribute in the DI component to describe an event listener feels strange to me.
Using `\Closure` would be super generic (yes, the `Closure` class doesn't have the `Attribute` attribute, but we can still read it so this is still fine to me as a convention)

Reply:
I hesitated  between creating a class for this attribute or a "keyword". `\Closure::class` seems good to me since we add an attribute on a closure.